lib/groonga-query-log/command/verify-server.rb

```diff
@@ -50,7 +52,7 @@ def run(command_line, &callback)
             end
           end
         end
-        same
+        return same, verifier.n_execute_commands
```

Comment:
We should not change the return value of this.
This is also used in `bin/groonga-query-log-verify-server`.

Reply:
I see.
I reverted this change.